docker-compose-production.yml

```diff
@@ -163,7 +167,39 @@ services:
         - Config__RabbitMQ__Password=${App__RabbitMQ__Password}
     networks:
         rabbitmq-network:
```

Comment:
this one doesnt use mssql network anymore? Or did you remove this by accident?

Reply:
Added again. Thanks for mentioning.

Reply:
In the future we should change this as the jobscheduler is now communicating internally, this should be externally. But this is out of scope of this PR.